controllers/users.js

```diff
@@ -602,6 +601,31 @@ const setInDiscordScript = async (req, res) => {
   }
 };
 
+/**
+ * Patch Update user nickname
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const changeUserNickname = async (req, res) => {
+  try {
+    const type = req.body.type;
+    const { user } = await userQuery.fetchUser({ userId: req.params.userId });
+
+    const { discordId, username: userName } = user;
```

Comment:
Just wanted to check that we are confident that the user object will have `discordId` present once we reach this controller function otherwise the API will fail